packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -351,4 +355,18 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
     }
     // else let the sandbox continue so customers can revert their changes
   };
+
+  private printSandboxNameInfo = async (optionalName?: string) => {
+    const sandboxBackendId = await this.backendIdSandboxResolver(optionalName);
+    const sandboxName =
+      BackendIdentifierConversions.toStackName(sandboxBackendId);
+    this.printer.log(
+      format.bold(`Starting sandbox for ${format.command(sandboxName)}`)
```

Comment:
We shouldn't reuse format.command for non-commands. Maybe we need format.highlight? (it could format to the same thing for now, but would allow us to change them independently in the future)

Reply:
updated